core/common/src/main/java/alluxio/wire/MasterInfo.java

```diff
@@ -9,11 +9,10 @@
  * See the NOTICE file distributed with this work for information regarding copyright ownership.
  */
 
-package alluxio.master.meta;
-
-import alluxio.wire.Address;
+package alluxio.wire;
```

Comment:
Please explain the reason why you move the package?

Reply:
Maybe we need create another MasterInfo in wire package?

Reply:
I will change that.